hack/generator/pkg/astbuilder/builder_test.go

```diff
@@ -19,20 +18,20 @@ func asplode(l, r dst.Expr) dst.Expr {
 }
 
 func TestReduceZero(t *testing.T) {
-	g := gomega.NewWithT(t)
+	g := NewWithT(t)
 
 	g.Expect(func() { Reduce(asplode) }).To(PanicWith("must provide at least one expression to reduce"))
 }
 
 func TestReduceOne(t *testing.T) {
-	g := gomega.NewWithT(t)
+	g := NewWithT(t)
 
 	expr := &dst.BadExpr{}
 	g.Expect(Reduce(asplode, expr)).To(BeIdenticalTo(expr))
 }
 
 func TestJoinOr(t *testing.T) {
-	g := gomega.NewWithT(t)
+	g := NewWithT(t)
```

Comment:
I’d say we should use either `gomega.NewWithT` or `NewGomegaWithT` but not `NewWithT` or `gomega.NewGomegaWithT`

Reply:
Can we use `gomega.NewWithT(t)` while still having the import in the current scope? Being able to write the assertions without the `gomega.` prefix is almost essential.

Reply:
I've been moving to `NewWithT()` because the older `NewGomegaWithT()` because the older one is deprecated:

``` go
// GomegaWithT is deprecated in favor of gomega.WithT, which does not stutter.
type GomegaWithT = WithT
```

No, it looks like we can't. So, moving to `gomega.NewWithT()` also requires moving to

``` go
g.Expect(Reduce(asplode, expr)).To(gomega.BeIdenticalTo(expr))
```

Which is a bit suboptimal.

Reply:
Didn't we end up on using `NewGomegaWithT` in the meeting?

Reply:
Yes, we did.

Reply:
I've updated `builder_test.go` which is the only test file in this PR to use `NewGomegaWithT()`; I'll update other test files as I touch them (or, if I get a momement, I'll do a bulk cleanup PR.)